authorization/storage_authorization.go

```diff
@@ -374,14 +601,19 @@ func authorizationsPredicateFn(f influxdb.AuthorizationFilter) kv.CursorPredicat
 	}
 
 	if f.Token != nil {
-		exp := *f.Token
+		token := *f.Token
+		allHashes, _ := s.hasher.AllHashes(token) // on error, allHashes is empty and we'll ignore hashedToken
```

Comment:
But shouldn't we log the error?  As a clue for debugging?

Reply:
`authorization.Store` didn't have access to a logger until a few minutes ago. I'll add logging now.